RetailCoder.VBE/API/VBA/ParserState.cs

```diff
@@ -10,14 +10,15 @@
 using Rubberduck.Parsing.VBA;
 using Rubberduck.UI.Command.MenuItems;
 using Rubberduck.Parsing.Symbols;
-using Rubberduck.VBEditor.SafeComWrappers.VB.VBA;
+using Rubberduck.VBEditor.SafeComWrappers.VB;
+using Rubberduck.VBEditor.SafeComWrappers.VB.Abstract;
 
 namespace Rubberduck.API.VBA
 {
     [ComVisible(true)]
     public interface IParserState
     {
-        void Initialize(Microsoft.Vbe.Interop.VBE vbe);
+        void Initialize(object vbe);
```

Comment:
Why not `SafeComWrappers.VB.Abstract.IVBE`?

Reply:
The function takes a raw COM object, then wraps it into an IVBE - previously this was a VBA VBE, now uses a factory method to remove the dependency. There's no interface shared between VBA and VB6 COM interop types, hence object.

Reply:
@mansellan seems like a good place to add a "why" comment to help future readers, then

Reply:
will do :-)

Reply:
Gah, just noticed the namespace - of course it takes an interop VBE object; that method is meant to be invoked by VBA/VB6 code! Good call with the factory.